test/js/service/pensize/PenSizeServiceTest.js

```diff
@@ -56,7 +56,7 @@ describe("PenSize test suite", function() {
 
     penSizeService.init();
     // MAX_VALUE is 4
```

Comment:
nit: update the comment here MAX_VALUE is 32

and update the name of the test "skips invalid value (outside of [1, 32])"